tests/integration/test_fim/test_registry/test_recursion_level/test_recursion_level_registry.py

```diff
@@ -3,17 +3,14 @@
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 import os
```

Comment:
Please, add one blank line between the comment and the first import.
```suggestion

import os
```